git_wrapper/branch.py

```diff
@@ -42,8 +48,8 @@ def to_hash(self, branch_name, hash_):
 
         self.logger.debug("Successfully rebased branch %s to %s", branch_name, hash_)
 
-    def abort(self):
-        """Abort a rebase."""
+    def rebase_cancel(self):
```

Comment:
Putting the verb first does sound better! I was thinking about trying to make it easier to remember how to cancel operations with a single word, because when I was working on the commit.py stuff there were 3 different verbs to cancel an operation ("abort" applying a patch, "revert" a commit, "reverse" applying a diff) which gets confusing. Probably it is better to stick to the proper git terminology though, to avoid another kind of confusion.